cpp/cmake/thirdparty/get_libcudacxx.cmake

```diff
@@ -17,8 +17,9 @@ function(find_and_configure_libcudacxx)
   include(${rapids-cmake-dir}/cpm/libcudacxx.cmake)
 
   rapids_cpm_libcudacxx(
-    BUILD_EXPORT_SET cudf-exports INSTALL_EXPORT_SET cudf-exports PATCH_COMMAND patch
-    --reject-file=- -p1 -N < ${CUDF_SOURCE_DIR}/cmake/libcudacxx.patch || true
+    BUILD_EXPORT_SET cudf-exports
+    INSTALL_EXPORT_SET cudf-exports PATCH_COMMAND patch --reject-file=- -p1 -N <
```

Comment:
I'm not familiar with the syntax used in `cmake` files. Should there be a `\` at the end to indicate continuation?

Reply:
No idea, this is a style fix when the checker version changed. Merged trunk so these changes are gone from the PR.